package.json

```diff
@@ -194,20 +195,27 @@
     "node-restify": "^0.2.1",
     "pg": "^4.5.5",
     "pg-hstore": "^2.3.2",
+    "plotly.js": "^1.30.0",
     "postcss": "^5.0.19",
     "query-string": "^4.2.3",
     "ramda": "^0.21.0",
     "react": "^15.5.1",
     "react-ace": "^3.4.1",
     "react-dd-menu": "^2.0.2",
+    "react-dnd": "^2.5.1",
+    "react-dnd-html5-backend": "^2.5.1",
     "react-dom": "^15.5.1",
     "react-immutable-proptypes": "^1.7.1",
+    "react-plotly-dnd-editor": "^0.10.0",
```

Comment:
it looks like some files were copied over here from this project, so can we remove this dependency?

Reply:
yep, don't need this, I'll take it out